cookiecutter.json

```diff
@@ -2,20 +2,15 @@
   "author": "<INSERT AUTHOR HERE>",
   "author_email": "<INSERT EMAIL HERE>",
   "github_url": "If you have already made a github repo to tie the project to place it here, otherwise update in setup.py later.",
-  "adapter_name": "My Adapter",
-  "directory_name": "{{ cookiecutter.adapter_name.lower().replace(' ','') }}",
-  "project_name": "dbt-{{ cookiecutter.directory_name }}",
-  "description": "the {{ cookiecutter.directory_name }} adapter plugin for dbt.",
+  "adapter_name": "What is name of your database in CamelCase? ex. MyAdapter",
+  "project_name": "dbt-{{ cookiecutter.adapter_name.lower() }}",
   "is_sql_adapter": [
     "true",
     "false"
   ],
```

Comment:
This is definitely better than before, and I'm glad `true` is the default. We owe people an explanation of what this means. I think our best bet is write more about this in the companion docs, and link to it from here